example/properties_directory/structural/inchi.json

```diff
@@ -0,0 +1,4 @@
+{
+    "name": "inchi",
+    "value": "1S/Si2/c1-2"
```

Comment:
This seems like an "inchi" for fcc Si. Should we have it for something else more meaningful? Same for the inchi_key below

Reply:
We can do a small molecule like benzene

Reply:
`inchi = 1S/C6H6/c1-2-4-6-5-3-1/h1-6H`
`inchi_key = UHOVQNZJYSORNB-UHFFFAOYSA-N`